### PR TITLE
+ ruby28.y: add right hand assignment

### LINKED ISSUE #668: ruby/ruby@1b2d351
https://github.com/ruby/ruby/commit/1b2d351b216661e03d497dfdce216e0d51474664

### PR DESCRIPTION
**Pull request by palkan**
Closes #668.

### CODE REVIEW

**Comment by palkan** [comment]
Here is an example usage: https://github.com/ruby-next/ruby-next/pull/38

**Comment by iliabylich** [comment]
@palkan Could you merge https://github.com/whitequark/parser/pull/676 first please? There are conflicts and it's better to merge PRs one by one

**Comment by palkan** [comment]
@iliabylich Here is a dual PR for the _next_ grammar: https://github.com/ruby-next/parser/pull/4/files.

(I switched to using a separate grammar file for Ruby Next, which is based on the current master's `ruby28.y`. Endless def (#676) has been already merged https://github.com/ruby-next/parser/pull/3)

**Comment by iliabylich** [comment]
I don't follow how's it related to #676 and this PR. They should be merged one by one and #676 is still open.

Could you rebase #676 and mark PR as "ready or review"? If "merge" button is locked for you just LMK and I'll merge it by myself.

**Comment by palkan** [comment]
> I don't follow how's it related to #676 and this PR. 

Sorry, I thought you meant merging into my fork [here](https://github.com/whitequark/parser/pull/676#issuecomment-617941991), since I don't have merge permissions here.

**Comment by iliabylich** [comment]
^ That's what I was talking about 😄 Could you rebase please?

**Comment by palkan** [comment]
Done!

File: doc/AST_FORMAT.md [hunk 1]
```diff
@@ -730,6 +730,28 @@ Format:
...
+
+#### Multiple right-hand assignment
+
+Format:
+
+~~~
+(mrasgn (send (int 13) :divmod (int 5)) (mlhs (lvasgn :a) (lvasgn :b)))
+"13.divmod(5) => a,b"
+ ~~~~~~~~~~~~~~~~~~~ expression
+              ^^ operator},
```
Comment (iliabylich) [inline doc/AST_FORMAT.md:752]: `},` copy-paste artifact?

Comment (palkan) [reply doc/AST_FORMAT.md]: Looks so. Fixed.

**Comment by iliabylich** [comment]
Thanks!

**Pull request merged**